src/view/components/cpx/CpxSimulator.tsx

```diff
@@ -29,7 +29,7 @@ interface IState {
     cpx: ICpxState;
     play_button: boolean;
 }
-interface IMyProps {
+interface IProps {
```

Comment:
I don't think you need to define children as a prop, as it already available. So even if you pass empty props here, in your component this.props.children would still be available

Reply:
Yes, i'm making the change to the existing codebase